CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
     Changes that have landed in master but are not yet released.
     Click to see more.
   </summary>
+</details>
```

Comment:
@akleiner2 can you set up the `Minor` and `Patch` headings for the next unreleased group so others dont have to 🙏